.github/workflows/cloudflare.yml

```diff
@@ -0,0 +1,48 @@
+name: CloudFlare
+
+# Trigger the workflow when:
+on:
+  # A push occurs to one of the matched branches.
+  push:
+    branches: [master]
+  # Or when a pull request event occurs for a pull request against one of the
+  # matched branches.
+  pull_request:
+    branches: [master]
+
+jobs:
+  publish:
+    # This name appears in GitHub's Checks API.
+    name: publish
+    # Do not trigger job for dependency update bot.
+    if: github.actor != 'renovate[bot]'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Set up Node.js 18
+        uses: actions/setup-node@v3.5.0
+        with:
+          node-version: "18.x"
+          cache: yarn
+      - run: yarn install --frozen-lockfile
+      - run: yarn build-preview
+      - name: Publish to Cloudflare Pages
+        # Id is needed to access output in a next step
+        id: cloudflare
+        uses: cloudflare/pages-action@v1
+        with:
+          apiToken: ${{ secrets.CLOUDFRONT_API_TOKEN }}
+          accountId: ${{ secrets.CLOUDFRONT_ACCOUNT_ID }}
+          # Project name created via CloudFlare dashboard
+          projectName: explorer
+          directory: build
+      # On a subsequent run the original comment will be updated.
+      - name: Add Comment to PR
+        uses: mshick/add-pr-comment@v2
+        with:
+          message: |
+            ✅ Preview URL: ${{ steps.cloudflare.outputs.url }}
```

Comment:
Could we make it more verbose and similar to existing CloudFlare Pages comment?

Here is the source code of a CloudFlare Pages comment:
```
## Deploying with &nbsp;<a href="https://pages.dev"><img alt="Cloudflare Pages" src="https://user-images.githubusercontent.com/23264/106598434-9e719e00-654f-11eb-9e59-6167043cfa01.png" width="16"></a> &nbsp;Cloudflare Pages

<table><tr><td><strong>Latest commit:</strong> </td><td>
<code>f4e5b69</code>
</td></tr>
<tr><td><strong>Status:</strong></td><td>&nbsp;✅&nbsp; Deploy successful!</td></tr>
<tr><td><strong>Preview URL:</strong></td><td>
<a href='https://7d66dd1d.oasis-explorer.pages.dev'>https://7d66dd1d.oasis-explorer.pages.dev</a>
</td></tr>
<tr><td><strong>Branch Preview URL:</strong></td><td>
<a href='https://mz-cloudflare.oasis-explorer.pages.dev'>https://mz-cloudflare.oasis-explorer.pages.dev</a>
</td></tr>
</table>

[View logs](https://dash.cloudflare.com/?to=/:account/pages/view/oasis-explorer/7d66dd1d-4729-468c-a872-811458a62ed8)
```

I think we can omit the "View logs" and "Branch Preview URL" parts since we don't have those.

But we could keep the "Deployed with Cloudflare Pages" and the first 3 rows of the table?